docs/src/tutorials/tutorial_2.md

```diff
@@ -407,12 +352,12 @@ It's invocation in the code looks like this:
 
 ```julia
 ...
-Action(:same, :info, info_box),
+info = Object(info_box)
 ...
 ```
 
 > **NOTE:** The function for `info_box` is a little different!
-> Each `Action` inside of the `javis` function automatically receives three objects being a `Video` object, which was previously defined outside of the `javis` function, the `Action` object, and the current frame number. 
+> Each `Action` inside of the `render` function automatically receives three objects being a `Video` object, which was previously defined outside of the `render` function, the `Object`, and the current frame number. 
```

Comment:
It should be `Object` and it's not part of the `render` function but instead defined before it in v0.3

Reply:
Is my rephrasing better or how should I better word it?